sources/packages/backend/libs/integrations/src/esdc-integration/fed-restriction-integration/fed-restriction-processing.service.ts

```diff
@@ -259,7 +259,7 @@ export class FedRestrictionProcessingService {
           this.logger.log(
             `Generating ${insertedRestrictionsIDs.length} notification(s).`,
           );
-          await this.studentRestrictionService.createNotifications(
+          await this.integrationStudentRestrictionService.createNotifications(
```

Comment:
`createNotifications` seems to be a responsibility of a service at `@sims/services` layer or maybe I am missing something.